src/components/ArticleView/AllArticles.jsx

```diff
@@ -0,0 +1,36 @@
+import ArticlePreviewCard from "./ArticlePreviewCard";
+import axios from "axios";
+import React, { useState, useEffect } from "react";
+
+const AllArticles = () => {
+  const [articlesAll, setArticlesAll] = useState([]);
+  React.useEffect(() => {
+    axios
```

Comment:
extract this out into a utils file which returns the fetched articles and then sets the articles state here

Reply:
I'm not sure how to do this. I've tried but I can't find a note which explains this.